tests/sharedtria/tria_zorder_02.cc

```diff
@@ -0,0 +1,83 @@
+// ---------------------------------------------------------------------
+// $Id: 3d_refinement_01.cc 31349 2013-10-20 19:07:06Z maier $
+//
+// Copyright (C) 2008 - 2015 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE at
+// the top level of the deal.II distribution.
+//
+// ---------------------------------------------------------------------
+
+
+// create a shared tria mesh and distribute with zorder scheme
+// Unlike tria_zorder_01, this test does not need p4est library
```

Comment:
If it is easy to adjust the test, then it's worthwhile doing because that ensures that the 1d case is also tested. In general, the more test coverage you can get, the better.